src/test/java/WebDriverSeleniumTests/GoogleSearchForWikiEdFirefox.java

```diff
@@ -0,0 +1,91 @@
+package WebDriverSeleniumTests;
```

Comment:
see my comments from Edge area- here is the same picture

Reply:
+